### DISCUSSION TITLE
How to mix spin species

### DISCUSSION THREAD

**Discussion by thefacemyer** [QuSpin/QuSpin#659] [Q&A]
I am curious if there is way in QuSpin to mix spins in the model. For example, how would one formulate a Heisenberg spin chain with sublattice A having spin moment m1 and sublattice B having spin moment m2?

**Comment by mgbukov** [comment]
That should be doable with either the tensor basis (but you can't use
symmetries there other than particle conservation) or I guess with the user
basis.
